Components/Order/OrderCancellation.php

```diff
@@ -127,7 +127,9 @@ public function cancelPlacedOrder($order)
 
             $this->orderUpdater->updateShopwareOrderStatusWithoutMail(
                 $order,
-                PaymentStatus::MOLLIE_PAYMENT_CANCELED
+                PaymentStatus::MOLLIE_PAYMENT_CANCELED,
+                $this->config->useOrdersApiOnlyWhereMandatory()
```

Comment:
please use variable that speaks for itself, its hard to read what is happening with that in here.
maybe you can reuse a function or class from the checkout process? theres a facade that also figures out if orders api is being used